src/components/GradeBookContentContainer.tsx

```diff
@@ -7,112 +7,113 @@ import GradeItemAccordion from './GradeItemAccordion';
 const ContentContainer = styled.div`
 	color: ${(props) => props.theme.colors.text[1]};
 	font-weight: bold;
-`;
+	padding: 50px 0px 50px 50px;
 
-const TitleRow = styled.div`
-	display: flex;
-	flex-direction: row;
-	flex-wrap: wrap;
-	width: 100%;
-	border-style: hidden hidden solid hidden;
-	color: ${(props) => props.theme.colors.gray[1]};
+	display: grid;
+	grid-template-columns:
+		minmax(75px, 1fr) minmax(200px, 5fr) minmax(200px, 5fr)
```

Comment:
Super minor, but is there a reason why px must be used instead of rem or em?

Reply:
I know sometimes having view boundaries described in px does make sense, but if it makes sense to do it in em, that is preferred.